bot/seasons/evergreen/game.py

```diff
@@ -198,42 +208,38 @@ def __init__(self, bot: Bot):
     async def get_pages(self, data: List[Dict[str, Any]]) -> List[Tuple[str, str]]:
         """Generate all game pages, do additional requests to IGDB API."""
         pages = []
-        for game in data:
-            pages.append(await self.create_page(game))
+        [pages.append(await self.create_page(game)) for game in data]
```

Comment:
What about 
```suggestion
        pages = [await self.create_page(game) for game in data]
```
Doesn't that work?